.github/workflows/main.yml

```diff
@@ -7,10 +7,10 @@ name: Main workflow
 on:
   pull_request:
     branches:
-      - main
+      - rel-1.5
   push:
     branches:
-      - main
+      - rel-1.5
```

Comment:
I think this should be something like `rel-*` such that it can be used for any release branch. What do you think?